src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -377,16 +380,16 @@ public virtual TagBuilder GenerateAntiForgery([NotNull] ViewContext viewContext)
             bool allowMultiple,
             object htmlAttributes)
         {
-            ICollection<string> ignored;
+            var currentValues = GetCurrentValues(viewContext, modelExplorer, expression, allowMultiple);
```

Comment:
sorry, n00bz0r. What's current values mean in this context? Are they the selected values?